### PR TITLE
[Network]`az network express-route peering create`: Support ipv6 address

### PR DESCRIPTION
**Pull request by msyyc**
**Description**

https://github.com/Azure/azure-cli/issues/16848
`az network express-route peering create`: Support ipv6 address
**Testing Guide**

test_network_express_route_ipv6_peering2

**History Notes**

[Component Name 1] BREAKING CHANGE: az command a: Make some customer-facing breaking change.
[Component Name 2] az command b: Add some customer-facing feature.

---

This checklist is used to make sure that common guidelines for a pull request are followed.

- [ ] The PR title and description has followed the guideline in [Submitting Pull Requests](https://github.com/Azure/azure-cli/tree/dev/doc/authoring_command_modules#submitting-pull-requests).

- [ ] I adhere to the [Command Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/command_guidelines.md).

- [ ] I adhere to the [Error Handling Guidelines](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md).

### CODE REVIEW

**Comment by yonzhan** [comment]
Network

File: src/azure-cli/azure/cli/command_modules/network/custom.py [hunk 1]
```diff
@@ -2632,7 +2632,7 @@ def create_express_route_peering(
         cmd, client, resource_group_name, circuit_name, peering_type, peer_asn, vlan_id,
         primary_peer_address_prefix, secondary_peer_address_prefix, shared_key=None,
         advertised_public_prefixes=None, customer_asn=None, routing_registry_name=None,
-        route_filter=None, legacy_mode=None):
+        route_filter=None, legacy_mode=None, ip_version='IPv4'):
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/custom.py:2635]: Where does 'ip_version' argument come from?

File: src/azure-cli/azure/cli/command_modules/network/commands.py [hunk 2]
```diff
@@ -694,7 +694,7 @@ def _make_singular(value):
     with self.command_group('network dns', network_dns_reference_sdk, resource_type=ResourceType.MGMT_NETWORK_DNS) as g:
         g.command('list-references', 'get_by_target_resources')
 
-    with self.command_group('network dns zone', network_dns_zone_sdk) as g:
+    with self.command_group('network dns zone', network_dns_zone_sdk, resource_type=ResourceType.MGMT_NETWORK_DNS) as g:
```
Comment (kairu-ms) [inline src/azure-cli/azure/cli/command_modules/network/commands.py:697]: Maybe it's better to fix this issue in another PR

Comment (msyyc) [reply src/azure-cli/azure/cli/command_modules/network/commands.py:697]: both are ok I think

File: src/azure-cli/azure/cli/command_modules/network/custom.py (hunk 1 above)
Comment (msyyc) [reply src/azure-cli/azure/cli/command_modules/network/custom.py:2635]: It is described in SDK and could also be found in _params.py

**Review by kairu-ms — APPROVED** [review]

**Pull request merged**